models/action.go

```diff
@@ -262,6 +264,11 @@ func (pc *PushCommits) ToApiPayloadCommits(repoLink string) []*api.PayloadCommit
 		if err == nil {
 			author_username = author.Name
 		}
+		committer_username := ""
```

Comment:
Ok, I was going by what was used for author username later in the file.  I will change that as well for consistency.